tests/zfs-tests/tests/functional/suid/Makefile.am

```diff
@@ -0,0 +1,14 @@
+include $(top_srcdir)/config/Rules.am
+
+pkgdatadir = $(datadir)/@PACKAGE@/zfs-tests/tests/functional/suid
+
+dist_pkgdata_SCRIPTS = \
+	cleanup.ksh \
+	setup.ksh
+
+pkgexecdir = $(datadir)/@PACKAGE@/zfs-tests/tests/functional/suid
+
+pkgexec_PROGRAMS = suid_write_to_suid suid_write_to_sgid suid_write_to_suid_sgid
+suid_write_to_suid_SOURCES = suid_write_to_suid.c
+sgid_write_to_sgid_SOURCES = sgid_write_to_sgid.c
+suid_sgid_write_to_suid_sgid_SOURCES = suid_sgid_write_to_suid_sgid.c
```

Comment:
You're going to need to either add `uid_write_to_file.h` to these sources, or add an `EXTRA_DIST = uid_write_to_file.h` line.  Otherwise it won't be included in the `make dist` tarball which is why you're seeing the TEST build failures.

Reply:
The header is now gone after integration with wrapper .ksh files.